### ISSUE TITLE
[book] is it ok that builtin criteria can't "imply" other criteria?

### ISSUE THREAD

**Issue by Gankra** [bholley/cargo-vet#72]
You can have your custom criteria imply the builtins (so you can have stronger criteria), but I don't currently see a way to have custom criteria be "implied by" the builtins (so you can have weaker criteria).

Unclear if this matters / is intentional, but it's a weird asymmetry I figured I'd mention in case you hadn't thought about it.

**Comment by bholley** [comment]
Good point. Would it be trivial in our current implementation to introduce a companion `implied-by` to support edges in the other direction? We could defer actually implementing it until someone needs it.
